default.nix

```diff
@@ -428,6 +428,12 @@ let
         '';
 
         updateClientDeps = pkgs.writeScript "update-client-deps" ''
+          #!${pkgs.runtimeShell}
```

Comment:
`writeShellScript`

Reply:
Sadly that doesn't work. I'm assuming because our nixpkgs is too old?

Reply:
Ugh. I was hoping https://github.com/input-output-hk/iohk-nix/pull/166 would be done by now.

Fine - please add a comment to change this when we update our nixpkgs.